src/button.component.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+export default function Button(props) {
+  const {
+    children,
+    disabled = false,
+    loading = false,
+    ...remainingProps
+  } = props;
+  const background =
+    disabled || loading ? "opacity-50 bg-secondary" : "bg-warning";
+  return (
+    <button
+      className={`mb-8 font-bold py-2 px-4 rounded ${background}`}
```

Comment:
One common thing I've seen with these kinds of components is to allow `className` to be passed in as a prop, too, so that custom styles can be applied.

Reply:
good idea. Should it override or append?